### PR TITLE
UCM: Fix heap corruption caused by ucp_set_event_handler()

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/ucm/util/sys.c [hunk 1]
```diff
@@ -282,6 +284,7 @@ void ucm_strerror(int eno, char *buf, size_t max)
 
 void ucm_prevent_dl_unload()
 {
+    static ucs_init_once_t init_once = UCS_INIT_ONCE_INITIALIZER;
```
Comment (hoopoepg) [inline src/ucm/util/sys.c:287]: unaligned ```=```?

**Review by hoopoepg — APPROVED** [review]

**Comment by yosefe** [comment]
@hoopoepg can you pls take a look?

**Review by hoopoepg — APPROVED** [review]

**Comment by hoopoepg** [comment]
it failed to locate io_demo.sh...
it seems infra issues

**Comment by yosefe** [comment]
bot:pipe:retest

**Pull request merged**